lib/v2/components/custom_dialog.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+const double padding = 20;
+const double avatarRadius = 40;
+
+/// A custom dialog with top icon that can be used in multiple screens
+class CustomDialog extends StatelessWidget {
```

Comment:
this is a very system wide component. This should be on its own PR

Reply:
Done 👍